src/services/logger/logger.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+import Pino from 'pino'
+const logger = Pino()
+
+logger.info(`Hi! Here is my first massage`)
+
+logger.setLevel = function setLevel (newLevel: any) {
```

Comment:
If the logger already has an option to change the level (logger.level = level), maybe we should just give up on this function. The setLevel should be replaced with logger.level = level.

Reply:
because the level logger defined in index.ts, 
I we will use logger.level, the event of change will not actiive